pom.xml

```diff
@@ -48,6 +48,7 @@
 		<maven.compiler.source>1.8</maven.compiler.source>
 		<maven.compiler.target>1.8</maven.compiler.target>
 
+		<jbtis.version>4.5.0.AM1-SNAPSHOT</jbtis.version>
```

Comment:
Looks like 2/3 of verifications passed, with just some test failure in org.fusesource.ide.projecttemplates.tests.integration and org.fusesource.ide.jmx.camel.tests.integration ... see https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/jbosstools-fuse-Pull-Request/31/console